pibooth/booth.py

```diff
@@ -282,6 +282,13 @@ def find_capture_event(self, events):
                 rect = self._window.get_rect()
                 if pygame.Rect(0, 0, rect.width // 2, rect.height).collidepoint(event.pos):
                     return event
+            if event.type == pygame.FINGERDOWN:
+                LOGGER.debug("Touch action dectected: %s", event)
```

Comment:
maybe logging of event in functions "find_.." could be systematic

Reply:
Indeed, I'll try to think of something systematic